### PR TITLE
Created DecisionCriteriaUI and applicable DecisionCriteriaUITest

### PR DESCRIPTION
**Pull request by Jerem-Dough**
This PR is addressing User-Story #115 

Contained are two new contributions, DecisionCriteriaUI.java and DecisionCriteriaUITest.java

The UI prompts users to enter the criteria they would like to be using. It requests them to then enter "done" when finished. After this, it will print the specific criterion they have entered, back to them. 

The associated testing file could be more robust, but it is a start.

### CODE REVIEW

**Comment by Jerem-Dough** [comment]
I was the only group member involved with this specific PR.

**Comment by jody** [comment]
> I was the only group member involved with this specific PR.

Without a co-contributor there is no peer assessment and review. Please ask a peer, in your team or not, to review this PR and add comments to this conversation.

**Review by jody — CHANGES_REQUESTED** [review]
1. The implementation class lacks basic javadoc-level comments. Maintainers need to know the intents.

2. The unit tests appear to address the utility methods within the class. Although these may satisfy for the use as a data class, there is no indication of testing (or how to test) the **primary functionality** of the class, which appears to involve "UI". You may want to investigate how to pass a Scanner, Writer, or OutputStream as a parameter. Then the test and the run-time can each pass different objects to satisfy the specific needs.

**Comment by Jerem-Dough** [comment]
Understood, can I talk to you further about this assignment tomorrow before or after our lecture?

I'll also find a classmate to help with contributions during this time.

File: decision_support/src/metrocs/DecisionCriteriaUI.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        String input;
+
+        System.out.println("Enter decision criteria. Type 'done' when you are finished:");
+
+        while (true) {
+            System.out.print("Enter criterion: ");
+            input = scanner.nextLine();
+
+            if (input.equalsIgnoreCase("done")) {
+                break;
```
Comment (jmaestas7) [inline decision_support/src/metrocs/DecisionCriteriaUI.java:36]: Be careful in this case, where the input might never be "done." We also need to provide that option so the user knows what the breaking case is

File: decision_support/src/metrocs/DecisionCriteriaUITest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    void testAddCriteria() {
+        manager.addCriteria("Cost");
+        manager.addCriteria("Quality");
+
+        List<String> criteriaList = manager.getCriteriaList();
+        assertEquals(2, criteriaList.size());
+        assertEquals("Cost", criteriaList.get(0));
+        assertEquals("Quality", criteriaList.get(1));
+    }
+
```
Comment (jmaestas7) [inline decision_support/src/metrocs/DecisionCriteriaUITest.java:26]: We are missing  test for the above comment. We need to verify via a unit test that the breaking phrase does work

**Pull request closed without merging**